src/main/java/com/marginallyclever/convenience/W3CColorNames.java

```diff
@@ -164,4 +169,16 @@ public class SVGColorNames {
     public static ColorRGB get(String name) {
         return map.get(name);
     }
+
+    /**
+     * @param match the color to matcch
+     * @return the name matching the given color, or null if no match.
+     */
+    public static String get(ColorRGB match) {
+        for(String key: map.keySet()) {
```

Comment:
Have you considerd doing it with `stream` ? Here is an example : https://www.baeldung.com/java-map-key-from-value#a-functional-approach

Reply:
Am I correct that stream would go through all items in the map?  My way it stops on the first hit.  slightly faster.

Reply:
no, stream won't go to the end of the iteration, this is why they are built for.